### PR TITLE
SA-1540: UX Feedback for Report Builder

### PR DESCRIPTION
**Pull request by PatrickSison**
### What Changed
- [x]     Typeahead can swap out icon from search
- [x]         Timezone typeahead uses clock icon
- [x]     Increase column width of columns of bounce reasons table (and other tables in the tabs)
- [x]     Add sortable prop to group by table
- [x]     Shift the group by table by -1 px to avoid double line betweeen header (with select) and table
- [x]     Add border to legend circles in tooltip (see health score page)
- [x]     add legend circle to aggregates table at bottom
- [x]     Remove deprecated flag on count_rendered and count_initial_rendered
- [x]         Change label and description from Rendered to Opens

### How To Test
 - Verify that all tests are still valid
 - Verify that the the visual updates are in place.

### CODE REVIEW

File: src/pages/reportBuilder/components/tabs/BounceReasonsTable.js [hunk 1]
```diff
@@ -27,11 +27,11 @@ const filterBoxConfig = {
...
 const columns = [
-  { label: 'Count %', sortKey: 'count_bounce' },
-  { label: 'Classification', sortKey: 'classification_id' },
+  { label: 'Count %', minWidth: '90px', sortKey: 'count_bounce' },
+  { label: 'Classification', minWidth: '900', sortKey: 'classification_id' },
   { label: 'Category', sortKey: 'bounce_category_name' },
-  { label: 'Reason', width: '45%', sortKey: 'reason' },
-  { label: 'Domain', sortKey: 'domain' },
+  { label: 'Reason', minWidth: '1000', sortKey: 'reason' },
+  { label: 'Domain', minWidth: '90px', sortKey: 'domain' },
```
Comment (kchu1337) [inline src/pages/reportBuilder/components/tabs/BounceReasonsTable.js:34]: Having a set width made last last columns off the screen when you set the screen width smaller.

File: src/pages/reportBuilder/components/Legend.js [hunk 2]
```diff
@@ -13,6 +13,8 @@ function renderMetric(metric, uniqueLabel, removeMetric) {
           display="inline-block"
           height="16px"
           width="16px"
+          border="1px solid" // todo, yuck
```
Comment (kchu1337) [inline src/pages/reportBuilder/components/Legend.js:16]: You could export the dot as it's own component and have props for color, size, and positioning.

Comment (kchu1337) [reply src/pages/reportBuilder/components/Legend.js:16]: Maybe even a general component under src/components.

Comment (PatrickSison) [reply src/pages/reportBuilder/components/Legend.js:16]: 👍

File: src/pages/reportBuilder/components/tabs/BounceReasonsTable.js (hunk 1 above)
Comment (PatrickSison) [reply src/pages/reportBuilder/components/tabs/BounceReasonsTable.js:34]: Yeah, with smaller widths, we want the table to become horizontally scrollable.

**Pull request merged**